rspace/src/main/scala/coop/rchain/rspace/history/KeySegment.scala

```
@@ -0,0 +1,55 @@
+package coop.rchain.rspace.history
+
+import scodec.bits.ByteVector
+
+import scala.annotation.tailrec
+
+final case class KeySegment(value: ByteVector) {
+  require(value.size <= 127)
```

Please always specify for such checks (assert and require) the second parameter - message to show to a user in case requirement is not met. Otherwise if require fails, it fails silently, it makes debugging hard.